test/internet/test-net-blocklist.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
This test should be in `test/parallel`. the `test/internet` subset are tests that require active access to the internet to run correctly. As far as I can see, this does not.

Also, the test should include validation of error cases. For instance, verifying that the correct/expected errors are thrown when the input to fromJSON is invalid.